virttest/utils_test/libvirt.py

```diff
@@ -2878,6 +2878,15 @@ def create_scsi_disk(scsi_option, scsi_size="2048"):
             linux_modules.unload_module("scsi_debug")
         linux_modules.load_module("scsi_debug dev_size_mb=%s %s" %
                                   (scsi_size, scsi_option))
+
+        def _get_scsi_disk():
+            """attempt to get scsi disk multiple times until getting real one"""
+            scsi_disk = process.run("lsscsi|grep scsi_debug|awk '{print $6}'",
+                                    shell=True).stdout_text.strip()
+            return 'dev' in scsi_disk
+
+        utils_misc.wait_for(lambda: _get_scsi_disk(),
```

Comment:
If timeout is met, should we handle the error?  If that, we only need to invoke L 2891 when no timeout?

Reply:
utils_misc.wait_for () itself has limitation that it can not return the late iteration result. So actually here it works similar to time.sleep(), but maybe a little smarter return earlier if condition is met.
So line 2891 still is in need

Reply:
@dzhengfy 
1)here utils_misc.wait_for() just work similar to time.sleep,and let's lsssci output have 'dev' in their output
2)nothing else logic need be changed, so we don't need handle the error since the following code will handle that before

Reply:
I just feel there is little redundent codes, but acceptable.